snuba/datasets/storages/processors/replaced_groups.py

```diff
@@ -43,40 +51,64 @@ def process_query(self, query: Query, request_settings: RequestSettings) -> None
 
         set_final = False
         if project_ids:
-            final, exclude_group_ids, replacement_types = get_projects_query_flags(
-                list(project_ids), self.__replacer_state_name,
+            flags: ProjectsQueryFlags = get_projects_query_flags(
```

Comment:
the PR does not test the change to this query processor. Please let's cover the new case
https://github.com/getsentry/snuba/blob/master/tests/datasets/storages/processors/test_replaced_groups.py